stock_picking_accounting_date/models/stock_picking.py

```diff
@@ -18,7 +18,8 @@ def _compute_show_accounting_date(self):
             pick.show_accounting_date = False
             if pick.picking_type_code not in ("incoming", "outgoing"):
                 continue
-            if pick.move_ids.with_company(pick.company_id).product_id.filtered(
+            products = pick.move_ids.with_company(pick.company_id).product_id
+            if products.filtered(
                 lambda x: x.detailed_type == "product" and x.valuation == "real_time"
-            ):
+            ) and not products.filtered(lambda x: x.detailed_type == "consu"):
```

Comment:
This change looks incorrect.  `show_accounting_date` should be true when the picking involves both stockable products and consumables.